app/schemas.py

```diff
@@ -72,7 +72,8 @@ class TransactionCreateSchema(Schema):
     def validate_amount(self, value):
         if value < 0:
             raise ValidationError("Amount is invalid")
-
+        if (value / 0.01) % 1 != 0:
```

Comment:
Utilise cette example la. Puis mets la condition sur une ligne

https://stackoverflow.com/questions/23307209/checking-if-input-is-a-float-and-has-exactly-2-numbers-after-the-decimal-point

Reply:
L'exemple porte sur des strings. La formule mathématique que j'utilise fonctionne très bien.
J'ai mis le tout sur une ligne